app/controllers/two_factor_authentication/otp_verification_controller.rb

```diff
@@ -45,13 +45,22 @@ def create
     def handle_valid_confirmation_otp
       assign_phone
       track_mfa_added
+      annotate_valid_recaptcha_phone_assessment
```

Comment:
in my mind, it seems like the recaptcha assesment is directly linked to the MFA adding, what was the motivation behind the separate events? (like in my mind it seems easier to add 3 properties to 2 events, instead of create the new event?)

Reply:
I'm not totally sure I follow the feedback here. Is it that `recaptcha_assessment_annotated` is a dedicated analytics event, but we could incorporate it into an existing event? I think that could make sense, yeah. It'd be nice to have some semi-standardized format for including annotation metadata on events... maybe a `recaptcha_annotation` hash or something.

Reply:
Updated in dcc31d13d9 to include annotation in existing events.